NOTICE

```diff
@@ -1,5 +1,3 @@
-Copyright 2022
-
```

Comment:
Why should this be removed?

Reply:
I think it's because Volvo isn't mentioned in the context so it's an incomplete statement. Was asked by The Council to remove it.

Reply:
Makes sense!